Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```
@@ -213,7 +213,7 @@ configuration:
   required: false
   type: 13
   section: Connect
-description: Manage Palo Alto Networks Firewall and Panorama. For more information see Panorama documentation.
+description: Manage Palo Alto Networks Firewall and Panorama. Use this pack to manage Prisma Access through Panorama. For more information see Panorama documentation
```

```suggestion
description: Manage Palo Alto Networks Firewall and Panorama. Use this pack to manage Prisma Access through Panorama. For more information see Panorama documentation.
```
does this change meet the DoD of "adding a description to the integration that refers to SASE with an explanation of when each should be used"?

---

Yes